i18n/i18n.jsx

```diff
@@ -126,9 +129,15 @@ const languages = {
     tr: {
         value: 'tr',
         name: 'Türkçe',
-        order: 10,
+        order: 11,
```

Comment:
Why add the new one above the old one?

Reply:
This section should be (sorry if I made a mistake in the ticket)

```
const languages = {
    de: {
        value: 'de',
        name: 'Deutsch',
        order: 0,
        url: de,
    },
    en: {
        value: 'en',
        name: 'English',
        order: 1,
        url: '',
    },
    es: {
        value: 'es',
        name: 'Español',
        order: 2,
        url: es,
    },
    fr: {
        value: 'fr',
        name: 'Français',
        order: 3,
        url: fr,
    },
    it: {
        value: 'it',
        name: 'Italiano',
        order: 4,
        url: it,
    },
    ja: {
        value: 'ja',
        name: '日本語',
        order: 15,
        url: ja,
    },
    ko: {
        value: 'ko',
        name: '한국어 (Alpha)',
        order: 12,
        url: ko,
    },
    nl: {
        value: 'nl',
        name: 'Nederlands (Alpha)',
        order: 5,
        url: nl,
    },
    pl: {
        value: 'pl',
        name: 'Polski (Alpha)',
        order: 6,
        url: pl,
    },
    'pt-BR': {
        value: 'pt-BR',
        name: 'Português (Brasil)',
        order: 7,
        url: ptBR,
    },
    ro: {
        value: 'ro',
        name: 'Română',
        order: 8,
        url: ro,
    },
    ru: {
        value: 'ru',
        name: 'Pусский (Alpha)',
        order: 10,
        url: ru,
    },
    tr: {
        value: 'tr',
        name: 'Türkçe',
        order: 9,
        url: tr,
    },
    uk: {
        value: 'uk',
        name: 'Yкраїнська (Beta)',
        order: 11,
        url: uk,
    },
    'zh-TW': {
        value: 'zh-TW',
        name: '中文 (繁體)',
        order: 14,
        url: zhTW,
    },
    'zh-CN': {
        value: 'zh-CN',
        name: '中文 (简体)',
        order: 13,
        url: zhCN,
    },
};
```

Reply:
@jasonblais changes made